clients/python/llmengine/data_types.py

```diff
@@ -32,6 +32,7 @@ class GpuType(str, Enum):
     NVIDIA_TESLA_T4 = "nvidia-tesla-t4"
     NVIDIA_AMPERE_A10 = "nvidia-ampere-a10"
     NVIDIA_AMPERE_A100 = "nvidia-ampere-a100"
+    NVIDIA_AMPERE_A100E = "nvidia-ampere-a100e"
```

Comment:
can you check if this also needs to change on the spellbook side?

Reply:
> can you check if this also needs to change on the spellbook side?

asking because I know that was the case when we made changes for the inference framework

Reply:
i think spellbook won't need a change since this field will not be populated